crates/editor_ui/src/menu_toolbars.rs

```diff
@@ -462,14 +464,65 @@ pub fn top_menu(
+                    .stroke(stroke_default_color());
+                if ui
+                    .add(subscene_button)
+                    .on_hover_text("Open subscene")
+                    .clicked()
+                {
+                    let mut filedialog = egui_file::FileDialog::open_file(Some("assets".into()))
+                        .show_files_filter(Box::new(|path| {
+                            path.to_str().unwrap().ends_with(".scn.ron")
+                                || path.to_str().unwrap().ends_with(".gltf")
+                                || path.to_str().unwrap().ends_with(".glb")
+                        }))
+                        .title("Open Subscene (.scn.ron, .gltf, .glb)");
+                    filedialog.open();
+
+                    menu_state.subscene_dialog = Some(filedialog);
+                }
+
+                if let Some(subscene_dialog) = &mut menu_state.subscene_dialog {
+                    if subscene_dialog.show(ctx).selected() {
+                        if let Some(file) = subscene_dialog.path() {
+                            let mut path = file.to_str().unwrap().to_string();
+                            info!("path: {}", path);
+                            if path.starts_with("assets") {
+                                path = path.replace("assets", "");
+                                path = path.trim_start_matches("\\").to_string();
+                                path = path.trim_start_matches("/").to_string();
+
+                                if path.ends_with(".scn.ron") {
+                                    commands.spawn((PrefabBundle::new(&path), PrefabMarker));
+                                } else if path.ends_with(".gltf") || path.ends_with(".glb") {
+                                    commands.spawn((
+                                        SpatialBundle::default(),
+                                        GltfPrefab {
+                                            path,
+                                            scene: "Scene0".into(),
+                                        },
+                                        PrefabMarker,
+                                    ));
+                                } else {
+                                    error!("Unknown file type: {}", path);
+                                }
+                            }
+                        } else {
+                        }
+                    } else {
+                    }
+                }
+
                 let width = ui.available_width();
                 let distance = width / 2. - 40.;
                 ui.add_space(distance);
                 let play_button = egui::Button::new(to_richtext("▶", &sizing.icon))
                     .fill(SPECIAL_BG_COLOR)
                     .stroke(Stroke {
                         width: 1.,
-                        color: SELECTED_ITEM_COLOR,
+                        color: egui::Color32::DARK_GRAY,
```

Comment:
There was a cool reason for it to be dark blue:
1. More professional engines, like Unreal and Cryengine, use a nice color schema to differentiate important buttons. The play button is usually one of those
2. For color blind people, the default color used in those cases, green, is quite hard to see, being almost black, while this tone of blue is easily seen. 
3. Dark gray might be confusing with other lower level icons

Reply:
I really tried 20 seconds to unselect button. Select color its not good. Can we select another for it then?

Reply:
Sure we can use another color, I just didn't understand the problem, can you rephrase?

Reply:
I tried to deselect the Play button for about 20 seconds xDD

Reply:
https://github.com/rewin123/space_editor/pull/238

Reply:
<img width="262" alt="Captura de Tela 2024-03-05 às 7 11 43 PM" src="https://github.com/rewin123/space_editor/assets/14813660/8d6e54a9-992e-4268-995d-a9fb1f855302">

Uses unreal style